packages/app-store/zohocalendar/lib/CalendarService.ts

```diff
@@ -263,6 +265,37 @@ export default class ZohoCalendarService implements Calendar {
     );
   }
 
+  private async getUnavailability(
```

Comment:
Where did this method come from?

Reply:
As mention on the PR the existing implementation using the [free/busy](https://www.zoho.com/calendar/help/api/get-user-freebusy-details.html) endpoint was found insufficient for accurate availability checks:
 
- To use this endpoint, we need to enable the `free/busy` functionnality [on settings](Steps_to_enable_or_disable_free_busy_consideration)
- To correctly fetch busy events, you also need to [activate this functionality](https://help.zoho.com/portal/en/kb/calendar/create-and-manage-events/articles/free-busy-sharing#Steps_to_include_a_calendar_to_your_free_busy_schedule) for each event created. This is a complex mechanism.

Therefore, I've enhanced the `getAvailability` function to include checks for all events without differentiating between free and busy statuses.